pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -469,6 +476,8 @@ def _share_information_to_prevent_deadlock(self):
 
         self._sync_dir = sync_dirs[self.node_rank]
 
+        self._has_called_call_children_scripts = self.broadcast(self._has_called_call_children_scripts)
```

Comment:
why would we broadcast this? processes > 0 never spawn any more children!

Reply:
The value needs to be broadcasted so all rpocesses are aware Lightning created the processes. Without this, only rank 0 knows

Reply:
it's a little unforunate as we really only need to communicate intra-node from local_rank=0 to local_rank > 0, but here we're forced to go from global 0 -> all ranks to communicate this.